app/models/shipit/status_group.rb

```diff
@@ -9,7 +9,7 @@ def initialize(significant_status, visible_statuses)
       @statuses = visible_statuses
     end
 
-    delegate :success?, :state, to: :significant_status
+    delegate :pending?, :success?, :error?, :failure?, :state, to: :significant_status
```

Comment:
I don't see new calls to these methods, was it necessary to add the delegation? I'm not against it though.